sql/src/main/java/io/crate/metadata/information/InformationColumnsTableInfo.java

```diff
@@ -135,8 +136,8 @@ private static ColumnRegistrar columnRegistrar() {
                 () -> NestableCollectExpression.forFunction(ColumnContext::getOrdinal))
             .put(Columns.DATA_TYPE,
                 () -> NestableCollectExpression.forFunction(r -> r.info.valueType().getName()))
-            .put(Columns.COLUMN_DEFAULT,
-                () -> NestableCollectExpression.constant(null))
+            .put(Columns.COLUMN_DEFAULT, () -> NestableCollectExpression.forFunction(
+                r -> SymbolPrinter.INSTANCE.printUnqualified(r.info.defaultExpression())))
```

Comment:
wouldn't that raise a `NullPointerException` if a column does not have a `default` expression?

Can you also add a test ?

Reply:
yep, tests fail already.